compositor_integration_tests/src/tests/simple.rs

```diff
@@ -0,0 +1,66 @@
+use crate::{
+    compare_dumps, dump_output, input_dump_from_disk, output_dump_from_disk, CommunicationProtocol,
+    CompositorInstance, PacketSender,
+};
+use anyhow::Result;
+use serde_json::json;
+use std::time::Duration;
+
+pub fn run_simple_test(mut instance: CompositorInstance, update_dumps: bool) -> Result<()> {
+    instance.send_request(json!({
+        "type": "register",
+        "entity_type": "rtp_input_stream",
+        "transport_protocol": "udp",
+        "input_id": "input_1",
+        "port": 8004,
+        "video": {
+            "codec": "h264"
+        }
+    }))?;
+
+    instance.send_request(json!({
+        "type": "register",
+        "entity_type": "output_stream",
+        "output_id": "output_1",
+        "transport_protocol": "tcp_server",
+
+        "port": 8002,
+        "video": {
+            "resolution": {
+                "width": 1920,
+                "height": 1080,
+            },
+            "encoder_preset": "medium",
+            "initial": {
+                "id": "input_1",
+                "type": "input_stream",
+                "input_id": "input_1",
+            }
+        }
+    }))?;
+
+    instance.send_request(json!({
+        "type": "start",
+    }))?;
+
+    let packets_dump = input_dump_from_disk("8_colors_input.rtp")?;
+    let sender = PacketSender::new(CommunicationProtocol::Udp, 8004)?;
+    sender.send(&packets_dump)?;
+
+    let new_output_dump = dump_output(
+        8002,
+        CommunicationProtocol::Tcp,
```

Comment:
How would this work for UDP? For udp you need to start listening before you register output, but on the other hand this is blocking. You need to rethink API of those helpers.